build error

An error occurred while I was building xml2js：
``` 
D:\Code\node-xml2js>npm run build

> xml2js@0.4.23 build D:\Code\node-xml2js
> cake build

events.js:167
      throw er; // Unhandled 'error' event
      ^

Error: spawn coffee ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:232:19)
    at onErrorNT (internal/child_process.js:407:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
    at Function.Module.runMain (internal/modules/cjs/loader.js:745:11)
    at startup (internal/bootstrap/node.js:279:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:752:3)

Emitted 'error' event at:
    at Process.ChildProcess._handle.onexit (internal/child_process.js:238:12)
    at onErrorNT (internal/child_process.js:407:16)
    [... lines matching original stack trace ...]

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! xml2js@0.4.23 build: `cake build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the xml2js@0.4.23 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\jiawulin-pc\AppData\Roaming\npm-cache\_logs\2020-02-17T08_41_50_680Z-debug.log
````
I tried to execute the `coffee -cw -o lib src` command,It worked。
![image](https://user-images.githubusercontent.com/14244944/74638285-97b71300-51a6-11ea-8134-451c3546a4b0.png)

--- a/Cakefile
+++ b/Cakefile
@@ -1,7 +1,8 @@
 {spawn, exec} = require 'child_process'
 
 task 'build', 'continually build the JavaScript code', ->
-  coffee = spawn 'coffee', ['-cw', '-o', 'lib', 'src']
+  coffeeScript = if process.platform == 'win32' then 'coffee.cmd' else 'coffee'
+  coffee = spawn coffeeScript, ['-cw', '-o', 'lib', 'src']
   coffee.stdout.on 'data', (data) -> console.log data.toString().trim()
 
 task 'doc', 'rebuild the Docco documentation', ->

